### PR TITLE
tests: add test case with minimum not at index 0

### PR DESCRIPTION
**Pull request by vil02**
#4388 has updated testcases in `FindMinTest`, but for all test arrays the minimal value is at index 0. This PR fixes that.

@siriak, @lukasb1b please have a look.

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

File: src/test/java/com/thealgorithms/maths/FindMinTest.java [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
-    @MethodSource("provideStringsForIsBlank")
+    @MethodSource("inputStream")
     void numberTests(int expected, int[] input) {
         Assertions.assertEquals(expected, FindMin.findMin(input));
     }
 
-    private static Stream<Arguments> provideStringsForIsBlank() {
-        return Stream.of(Arguments.of(1, new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10}), Arguments.of(5, new int[] {5, 5, 5, 5, 5}), Arguments.of(0, new int[] {0, 192, 384, 576}), Arguments.of(-1, new int[] {-1, 2, 5, 10}), Arguments.of(-10, new int[] {-10, -9, -8, -7, -6, -5, -4, -3, -2, -1}));
+    private static Stream<Arguments> inputStream() {
+        return Stream.of(Arguments.of(1, new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10}), Arguments.of(5, new int[] {5, 5, 5, 5, 5}), Arguments.of(0, new int[] {0, 192, 384, 576}), Arguments.of(-1, new int[] {-1, 2, 5, 10}), Arguments.of(-10, new int[] {-10, -9, -8, -7, -6, -5, -4, -3, -2, -1}),
```
Comment (siriak) [inline src/test/java/com/thealgorithms/maths/FindMinTest.java:21]:
```suggestion
        return Stream.of(
            Arguments.of(1, new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10}),
            Arguments.of(5, new int[] {5, 5, 5, 5, 5}),
            Arguments.of(0, new int[] {0, 192, 384, 576}),
            Arguments.of(-1, new int[] {-1, 2, 5, 10}),
            Arguments.of(-10, new int[] {-10, -9, -8, -7, -6, -5, -4, -3, -2, -1}),
```
Can we do something like that?

Comment (vil02) [reply src/test/java/com/thealgorithms/maths/FindMinTest.java:21]: No, `clang-format` will complain. The reason is this:
https://github.com/TheAlgorithms/Java/blob/cada67b1eb58558b08049b84ee93a15de7867e4b/.clang-format#L52

Should I make another PR and set it to something like `100` or `80` and reformat all of the files?

Comment (siriak) [reply src/test/java/com/thealgorithms/maths/FindMinTest.java:21]: Hmm, I think no. There should be a better way than using ColumnLimit

**Review by siriak — APPROVED** [review]

**Review by lukasb1b — COMMENTED** [review]
Good Job seems like I overlookt that
